app/views/assignments/_list_manage.html.erb

```diff
@@ -7,77 +7,105 @@
     <div class='table'>
       <table>
         <thead>
-          <tr>
-            <th><%= t(:assignment_item) %></th>
-            <th><%= t(:due) %></th>
-            <th><%= t(:submission_status) %></th>
-          </tr>
+        <tr>
```

Comment:
I'm not going to review this part until you change the indentation back to how it was before.